searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java

```diff
@@ -0,0 +1,67 @@
+package org.jalau.at18.searchobject.middleware;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and property information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
+import org.jalau.at18.searchobject.common.logger.At18Logger;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.servlet.*;
+import javax.servlet.annotation.WebFilter;
+/**
+ * Filters are Java classes that implement the javax.servlet.Filter interface,
+ * and whose mission is to intercept requests before they reach the servlets
+ * and after, to perform certain operations.
+ * @author Sarai Alvarez
+ * @version 1.0
+ */
+@WebFilter(urlPatterns = "/emotionRecognition")
+
+public class EmotionRecognitionMiddleware implements Filter {
+    private static final Logger LOG = new At18Logger().getLogger();
```

Comment:
The metod `.getLogger()` is a static method, it couldn't be accesed in a static way `At18Logger.getLogger();` ?

Reply:
Is necessary keep it and cannot be removed, must be initialized.